### DISCUSSION TITLE
Is Responsible data analytics achievable?

### DISCUSSION THREAD

**Discussion by Blessingz1** [community/community#88712] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Can we integrate responsible measures in data analytics tools? I'm building a forex and stock analytics platform but need to make it responsible with accountability and regulations in all transactions.

**Comment by AlessioBugetti** [comment]


**Comment by Andrkenya** [comment, marked as answer]
This could be quite challenging to curate in the initial stages, but I believe its achievable.
- First, you have to acquire adequate knowledge regarding mixing the two aspects.
- This can be through extensive research in this duo.
- Formulate a structured plan to involve all stakeholders.
- Ensure the involved parties come to an agreement on the user requirements viable in line with developer efforts.
- Finally proceed with implementing the project.
- Perform continuous evaluation based on user reviews as you advance with the project.

But your query is significantly ideal in this AI-based analytics era where regulators might need such responsible platforms. Thanks.

**Reply by Blessingz1** [reply]
Thanks, @Andrkenya for your views which sound interesting. I believe they will be beneficial.
